src/targets/gpu/include/migraphx/gpu/context.hpp

```diff
@@ -180,16 +180,12 @@ struct context
 
     void from_value(const value& v)
     {
-        assert(v.contains("events"));
-        auto v_events = v.at("events");
-        assert(v_events.is_uint64());
-        std::size_t n_events = v_events.get_uint64();
+        auto v_events        = v.at("events");
+        std::size_t n_events = v_events.to<std::pair<std::string, std::size_t>>().second;
```

Comment:
Actually, you can write `v_events.without_key().to<std::size_t>()`. In PR #616, I updated `value` class so the `without_key()` is not necessary.